hotel_housekeeping/models/hotel_housekeeping.py

```diff
@@ -109,6 +114,8 @@ def room_done(self):
         @param self: object pointer
         """
         self.state = 'done'
+        if not self.quality:
+                raise ValidationError(_('Please update quality of work!'))
```

Comment:
Extra tab before raise statement

Reply:
@JayVora-SerpentCS  Removed extra spaces and extra tab from code.